packages/create-lz-oapp/src/config.ts

```diff
@@ -55,6 +55,13 @@ export const EXAMPLES: Example[] = [
                   directory: 'examples/oft-solana',
                   ref,
               },
+              {
+                  id: 'oft-solana-v2',
+                  label: 'OFT v2 (Solana)',
+                  repository,
+                  directory: 'examples/oft-solana-v2',
+                  ref,
+              },
```

Comment:
think we need to delete the one above since it got deleted

Reply:
I deleted this one as we are now calling v2 `oft-solana`.